lib/tasks/roles.rake

```diff
@@ -0,0 +1,12 @@
+namespace :roles do
+  desc "Creates the initial set of roles"
+  task initial_set: :environment do
+    ['admin', 'board_member'].each do |role|
```

Comment:
@nellshamrell @hollomancer @kylemh - Outside of just a standard logged in, authorized user, what roles might we want to have initially?  BTW, this can be changed at any time.

Here I have `admin` and `board_member`, but those are just placeholder ideas.  In general, I am thinking something like:

- all access (something like `super_admin`?)
- less access (something like `admin`?)
- the least amount of access (something like `board_member`?)

Thoughts on that?  Recommendations?

Reply:
Overall, this looks good.  I would perhaps do something like super_admin (all access), admin (less access), member (least access).  Do our board members need special permissions to the site?

Reply:
@nellshamrell - We actually don't need a role for a standard member.  Their auth privileges will be gated by Devise, through the `current_user` process.

In that same vein, we will not need a role for an ID.me verified veteran user, as that will be gated by an authorized user with a `user.verified` attribute of `true`.

So these roles only apply to people we want to give **additional** access to, on top of the standard member, or verified veteran member.

re: `board_member`, I was thinking something along the lines of where a board member could see certain stat pages that normal members could not see.  Whereas an admin would have access to those pages, plus some additional `write` privileges.  And then of course a super admin would have everything.

Those are just _examples_...trying to paint a picture.  We can set things to be however we want them.  

I'll create those three role for now, to get this PR merged, so we can move onto beta implementation.  Then we can update at any time.